src/core/nsfs.js

```diff
@@ -75,13 +76,14 @@ async function main(argv = minimist(process.argv.slice(2))) {
         }
         const http_port = Number(argv.http_port) || 6001;
         const https_port = Number(argv.https_port) || 6443;
+        const backend = (argv.backend);
```

Comment:
you can use `argv.backend` inline so this doesn't seem to be needed

Reply:
or better off set this with default already, so that the prints later will show the actual value:
```
const backend = argv.backend || (process.env.GPFS_DL_PATH ? 'GPFS' : '');
```

later when creating fs_config you can use the short field notation of `backend,` which stands for `backend: backend,` which is nice.

Reply:
done